crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -609,6 +609,7 @@ impl ExecutorState {
                 // SAFETY:
                 // - The caller ensures that we have permission to
                 // access the world data used by the system.
+                // - `is_exclusive` returned false
```

Comment:
is exclusive isn't checked in this function. You need to add another safety bullet to `spawn_system_task` and add a bullet the unsafe in `spawn_system_tasks`